lib/hashes.md

```diff
@@ -5,40 +5,40 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A Hash is similar to an Array but allows us to organize multiple lists together.
 
 # What are some examples of information that would be Hashes as opposed to some other data type?
 
-Replace me with your response...
+If you were making an errand list or maybe a packing list for a family, it would make more sense to use a Hash rather than another data type. You would be able to make specific lists tied to each type of errand or items needed for each family member.
 
 # How are Hashes and Arrays similar? How are they different?
 
-Replace me with your response...
+Both Hashes and Arrays can contain multiple elements that make up lists of data. The elements in an Array are addressed by using the indexed position in the Array starting at 0. Hashes on the other hand have keys and values. By using a Hash you can make more complex lists of data instead of being limited to a single list when using an Array.
```

Comment:
A slight pushback: Arrays _can_ contain lists inside them (ex: `arr = [1, 2, "cat", ["a", "b", "c"], "dog"]`). Hashes can also be kind of boring (ex: `h = {"a" => "apple", "b" => "banana"}`). I think the key _(heh, heh)_ difference -- which you outlined -- is how information is organized and retrieved inside each of these collections.